CONTRIBUTING.rst

```diff
@@ -2,38 +2,56 @@
 Contributing to Klein
 =====================
 
-Klein is an open source project that welcomes contributions of all kinds coming from the community.
-
-These include:
+Klein is an open source project that welcomes contributions of all kinds coming from the community, including:
 
 - code patches,
 - `documentation <http://klein.readthedocs.org/>`_ improvements,
 - `bug reports <https://github.com/twisted/klein/issues>`_,
 - reviews for `contributed patches <https://github.com/twisted/klein/pulls>`_.
 
-Here's some suggestions to make the contributing process much smoother for you:
+
+Getting started
+===============
+
+Here is a list of shell commands that will install the dependencies of Klein, run the test suite, and compile the documentation.
+
+::
+   pip install --user -r requirements.txt
```

Comment:
Tox makes a virtualenv, installs the requirements into it, and runs the tests. The only external dependency a user will need is `tox`, so maybe `pip install tox` will be better here.